### ISSUE TITLE
sudo tar -I lz4 -C /var -xvf /preloaded.tar.lz4: exit status 2

### ISSUE THREAD

**Issue by wuyd2** [kubernetes/minikube#6965]
centos7 Virtual Machine [Vmware]
[root@localhost ~]# minikube start  --image-repository=registry.cn-hangzhou.aliyuncs.com/google_containers    --vm-driver=none 
* minikube v1.8.1 on Centos 7.4.1708
* Using the none driver based on user configuration
* Using image repository registry.cn-hangzhou.aliyuncs.com/google_containers
* Running on localhost (CPUs=4, Memory=1494MB, Disk=100001MB) ...
* Downloading preloaded images tarball for k8s v1.17.3 ...
* OS release is CentOS Linux 7 (Core)
    > preloaded-images-k8s-v1-v1.17.3-docker-overlay2.tar.lz4: 9.86 KiB / 499.2E0309 08:35:00.433747   75580 config.go:71] Failed to preload container runtime Docker: extracting tarball: 
** stderr ** 
Error 68 : Unfinished stream 
tar: Child returned status 68
tar: Error is not recoverable: exiting now

** /stderr **: sudo tar -I lz4 -C /var -xvf /preloaded.tar.lz4: exit status 2
stdout:

stderr:
Error 68 : Unfinished stream 
tar: Child returned status 68
tar: Error is not recoverable: exiting now
, falling back to caching images
    > preloaded-images-k8s-v1-v1.17.3-docker-overlay2.tar.lz4: 169.52 KiB / 499* Preparing Kubernetes v1.17.3 on Docker 19.03.7 ...
    > preloaded-images-k8s-v1-v1.17.3-docker-overlay2.tar.lz4: 198.83 KiB / 499* Launching Kubernetes ... 
    > preloaded-images-k8s-v1-v1.17.3-docker-overlay2.tar.lz4: 200.18 KiB / 499* 
X Error starting cluster: init failed. output: "": /bin/bash -c "sudo env PATH=/var/lib/minikube/binaries/v1.17.3:$PATH kubeadm init --config /var/tmp/minikube/kubeadm.yaml  --ignore-preflight-errors=DirAvailable--etc-kubernetes-manifests,DirAvailable--var-lib-minikube,DirAvailable--var-lib-minikube-etcd,FileAvailable--etc-kubernetes-manifests-kube-scheduler.yaml,FileAvailable--etc-kubernetes-manifests-kube-apiserver.yaml,FileAvailable--etc-kubernetes-manifests-kube-controller-manager.yaml,FileAvailable--etc-kubernetes-manifests-etcd.yaml,Port-10250,Swap,SystemVerification": signal: segmentation fault
stdout:

stderr:

* 
* minikube is exiting due to an error. If the above message is not useful, open an issue:
  - https://github.com/kubernetes/minikube/issues/new/choose

**Comment by afbjorklund** [comment]
Do you have `lz4` installed ? Seems like tar installed half an image, maybe there should some sanity checks after unpacking... Not sure how you would disable the preload

**Comment by lienharc** [comment]
Had the same issue on arch linux and minikube with kvm2, `minkube delete && minikube start` solved the issue.

(For what it's worth, this was the output, before I recreated minikube):
```
Starting existing kvm2 VM for "minikube" ...
E0311 10:04:57.167478   11785 config.go:71] Failed to preload container runtime Docker: extracting tarball: 
** stderr ** 
tar: invalid option -- 'I'
BusyBox v1.29.3 (2020-02-20 00:24:17 PST) multi-call binary.

Usage: tar c|x|t [-hvokO] [-f TARFILE] [-C DIR] [-T FILE] [-X FILE] [--exclude PATTERN]... [FILE]...

Create, extract, or list files from a tar file

	c	Create
	x	Extract
	t	List
	-f FILE	Name of TARFILE ('-' for stdin/out)
	-C DIR	Change to DIR before operation
	-v	Verbose
	-O	Extract to stdout
	-o	Don't restore user:group
	-k	Don't replace existing files
	-h	Follow symlinks
	-T FILE	File with names to include
	-X FILE	File with glob patterns to exclude
	--exclude PATTERN	Glob pattern to exclude

** /stderr **: sudo tar -I lz4 -C /var -xvf /preloaded.tar.lz4: Process exited with status 1
stdout:

stderr:
tar: invalid option -- 'I'
BusyBox v1.29.3 (2020-02-20 00:24:17 PST) multi-call binary.

Usage: tar c|x|t [-hvokO] [-f TARFILE] [-C DIR] [-T FILE] [-X FILE] [--exclude PATTERN]... [FILE]...

Create, extract, or list files from a tar file

	c	Create
	x	Extract
	t	List
	-f FILE	Name of TARFILE ('-' for stdin/out)
	-C DIR	Change to DIR before operation
	-v	Verbose
	-O	Extract to stdout
	-o	Don't restore user:group
	-k	Don't replace existing files
	-h	Follow symlinks
	-T FILE	File with names to include
	-X FILE	File with glob patterns to exclude
	--exclude PATTERN	Glob pattern to exclude
, falling back to caching images
minikube-start: 🐳  Preparing Kubernetes v1.17.3 on Docker 19.03.6 ...
```

**Comment by tstromberg** [comment]
I believe this was fixed in v1.8.2. Do you mind confirming?

**Comment by tstromberg** [comment]
Confirmed that we now skip preload for non-VM machines:
https://github.com/kubernetes/minikube/pull/7015
